client/src/document/baseline-indicator.tsx

```diff
@@ -62,9 +63,12 @@ export function BaselineIndicator({ status }: { status: SupportStatus }) {
     LOCALIZED_BCD_IDS[locale] || LOCALIZED_BCD_IDS[DEFAULT_LOCALE]
   }`;
 
+  const low_date_range = status.baseline_low_date?.match(/^([^0-9])/)?.[0];
+  const low_date_before = ["<", "≤"].includes(low_date_range || "");
```

Comment:
What about gt/gte? (It's mentioned in the Jira ticket.)

Reply:
There isn't any in the BCD data, and I doubt there will be - it doesn't really make sense, so I decided to save some bytes and not implement the "after" string (but still support any kind of non-numeric character preceding the date so we don't get a NaN).

Reply:
Makes sense, but I would suggest updating the Jira issue description, maybe just striking the part about gt/gte.